### ISSUE TITLE
xdebug installation fails when php version is 7.1

### ISSUE THREAD

**Issue by uran1980** [laradock/laradock#2807]
**No valid packages found** error

**Comment by uran1980** [comment]
in file `laradock/php-fpm/Dockerfile` add this:

```
elif [ $(php -r "echo PHP_MAJOR_VERSION;") = "7" ] && [ $(php -r "echo PHP_MINOR_VERSION;") = "1" ] ; then \
    pecl install xdebug-2.9.8; \
```

Full block of code is looking like this fix the problem:

```
RUN if [ ${INSTALL_XDEBUG} = true ]; then \
  # Install the xdebug extension
  if [ $(php -r "echo PHP_MAJOR_VERSION;") = "5" ]; then \
    pecl install xdebug-2.5.5; \
  elif [ $(php -r "echo PHP_MAJOR_VERSION;") = "7" ] && [ $(php -r "echo PHP_MINOR_VERSION;") = "1" ] ; then \
    pecl install xdebug-2.9.8; \
  else \
    pecl install xdebug; \
  fi && \
  docker-php-ext-enable xdebug \
;fi
```

**Comment by jonpage999** [comment]
You probably don't have the latest code. This has been fixed in PR #2793
